### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by GhostEsso**
Hello **Code Reviewer :hand:,**
I hope you're doing well :sunglasses: !
In this Milestone, I have theses features
- Show books;
- Show people;
- Create person;
- Create book;
- ids;

Thank you for reviewing my **Pull Request** :pray:, stay **blessed** :star2:

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    student_or_teacher = gets.chomp.to_i
+    if student_or_teacher == 1
+      create_student
+    elsif student_or_teacher == 2
+      create_teacher
```
Comment (julie-ify) [inline app.rb:41]: - [ ] The app crashes when I try to create a teacher. Kindly fix that.

  image

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    unless (0...@people.length).include?(person_index)
+      puts "Can not add a record. Person #{person_index} doesn't exist"
+      return
+    end
+    person = @people[person_index]
+    print 'Date: '
+    date = gets.chomp.to_s
+    @rentals.push(Rental.new(date, book, person))
+    puts 'Rental created successfully'
+  end
```
Comment (julie-ify) [inline app.rb:117]: - [ ] The app crashes when I try to create a rental. Kindly fix it.

  image

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+      puts "Person with id=#{id} doesn't exist"
+      return
+    end
+    puts 'Rentals:'
+    selected.map { |rental| puts "Date: #{rental.date}, Book \"#{rental.book.title}\" by #{rental.book.author}" }
+  end
+
+  def run
+    prompt
+  end
```
Comment (julie-ify) [inline app.rb:133]: - [ ] As I could not create a rental, I could not explore this option, so kindly ensure it works as well.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @GhostEsso,

Good job so far!

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @GhostEsso👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Good job with basic console UI. ✔️
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**